test/node/preprocess/encoder.ts

```diff
@@ -58,3 +58,18 @@ describe('OneHot Encoder', () => {
     assert.deepEqual(bxCate.dataSync() as any, bx);
   });
 });
+
+describe('Label Encoder', () => {
+  it('encode', async () => {
+    const encoder = new LabelEncoder();
+    await encoder.init(x);
+    const xEncode = await encoder.encode(x);
+    assert.deepEqual(xEncode.dataSync(), xLabelEncode.dataSync());
```

Comment:
```suggestion
    assert.deepEqual(await xEncode.data(), await xLabelEncode.data());
```
There is no need to use `dataSync()` here.